### DISCUSSION TITLE
Bea Milz - Criando mapas com os pacotes tidyverse e geobr

### DISCUSSION THREAD

**Discussion by giscus** [beatrizmilz/blog-en#31] [General]
# Bea Milz - Criando mapas com os pacotes tidyverse e geobr

https://beamilz.com/posts/2020-07-27-criando-mapas-com-os-pacotes-tidyverse-e-geobr/pt/

**Comment by brunocantisano** [comment]
Nossa! Excelente material.
Ajudou bastante a minha pesquisa!
